CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.2]
+
+### Fixed
+
+- Replace `@metamask/ethjs-{contract,query}` packages with `@ethersproject` for EIP1193 compatibility ([#155](https://github.com/MetaMask/eth-token-tracker/pull/155))
```

Comment:
Since we are mentioning the dependency updates above, maybe we can mention the exact problem we are fixing?

``` suggestion
- `TokenTracker` now supports passing in an EIP-1193-compatible provider ([#155](https://github.com/MetaMask/eth-token-tracker/pull/155))